addon/components/basic-dropdown-content.ts

```diff
@@ -50,11 +52,11 @@ export default class BasicDropdownContent extends Component<Args> {
 
 
   get destinationElement(): Element | null {
-    return document.getElementById(this.args.destination);
+    return document.getElementById(this.args.destination) ?? getOwner(this).rootElement.querySelector?.(`[id="${this.args.destination}"]`);
```

Comment:
There's a little bit of code duplication with the `getOwner(this).rootElement` part. Not sure if it's worth extracting into util.